src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -172,6 +173,11 @@ public override int Run(AnalyzeOptions analyzeOptions)
         {
             Stopwatch stopwatch = null;
 
+            if (analyzeOptions.DisableTelemetry == true)
+            {
+                this.Telemetry = null;
```

Comment:
This worries me a bit.  I see that if it's null, we just don't add a certain logger to an aggregate logger.  That looks great.

But then a little bit later, we pass this thing into a caller.  Does that call flow lead to uses?  Is it already nullchecked?

Reply:
yes it is, 
in fact our application is checking if it is null to see enabled or not, 
if null, no Telemetry will be performed.

Reply:
Scott's right, you haven't gotten this change quite right, which is to *enable* telemetry collection by default but allow someone to override that via this argument.

Where is the code that makes telemetry work out of box with no special configuration?